### DISCUSSION TITLE
Right clicking on the random button then moving cursor away performs rewind action but shouldnt

### DISCUSSION THREAD

**Discussion by Loreos7** [ppy/osu#21521] [Ideas] [answered]
Should keep same behaviour like other buttons.

https://user-images.githubusercontent.com/86934170/205681429-baa4705c-4668-4b0f-824d-807254002fde.mp4

**Comment by smoogipoo** [comment, marked as answer]
Resolved by https://github.com/ppy/osu/pull/21523
